### PR TITLE
Feat/showroom manager create password

### PR DESCRIPTION
**Pull request by vincentdurufle**

### CODE REVIEW

File: Manager/ShowroomUserManager.php [hunk 1]
```diff
@@ -125,6 +125,42 @@ public function create(User $user): ?User
...
+     * @return User|null
+     */
+    public function createPassword(string $email, string $password): ?User
+    {
+        $params = [
+            'email'    => $email,
+            'password' => $password,
+        ];
+        $apiResponse = $this->apiProvider->request(
+            'PUT',
```
Comment (florianajir) [inline Manager/ShowroomUserManager.php]: use POST method instead (see comment in api project)

File: Manager/ShowroomUserManager.php [hunk 2]
```diff
@@ -125,6 +125,42 @@ public function create(User $user): ?User
...
+            '/api/showroom/users/password',
```
Comment (florianajir) [inline Manager/ShowroomUserManager.php]: `POST /api/showroom/users/{$email}/password`

File: Manager/ShowroomUserManager.php [hunk 3]
```diff
@@ -125,6 +125,42 @@ public function create(User $user): ?User
...
+                RequestOptions::QUERY       => $params,
+                RequestOptions::HTTP_ERRORS => false,
+            ]
+        );
+        $updated = null;
+        if ($apiResponse->getStatusCode() === Response::HTTP_OK) {
+            $updated = $this->deserialize($apiResponse);
+        } else {
+            $this->logger->error(
+                'UserManager::update unexpected status code',
```
Comment (stvalett) [inline Manager/ShowroomUserManager.php]: UserManager::CreatePassword unexpected status code

File: Manager/ShowroomUserManager.php [hunk 4]
```diff
@@ -133,13 +133,10 @@ public function create(User $user): ?User
...
-        $params = [
-            'email'    => $email,
-            'password' => $password,
-        ];
+        $params = ['password' => $password];
         $apiResponse = $this->apiProvider->request(
-            'PUT',
-            '/api/showroom/users/password',
+            'POST',
+            '/api/showroom/users/'.$email.'/password',
```
Comment (stvalett) [inline Manager/ShowroomUserManager.php]: Improvement, but not necessary = sprintf('/api/showroom/users/%s/password', $email);

**Pull request merged**